src/vcModals.cpp

```diff
@@ -656,10 +656,22 @@ void vcModals_DrawProjectInfo(vcState *pProgramState)
     if (ImGui::BeginChild("##infoModal", ImVec2(-1, -30), true))
     {
       if (udProjectNode_GetMetadataString(pProgramState->activeProject.pRoot, "information", &pInfo, "") == udE_Success)
```

Comment:
Is this still required? Won't `pProgramState->projectInfoTextures.infoStrings.length` be 0 when this returns false?

Reply:
actually i can remove this if completely. the for loop will do nothing if the array is empty